crates/egui/src/containers/panel.rs

```diff
@@ -339,10 +342,12 @@ impl SidePanel {
                 Stroke::NONE
             };
             // TODO(emilk): draw line on top of all panels in this ui when https://github.com/emilk/egui/issues/1516 is done
-            // In the meantime: nudge the line so its inside the panel, so it won't be covered by neighboring panel
-            // (hence the shrink).
-            let resize_x = side.opposite().side_x(rect.shrink(1.0));
-            let resize_x = ui.painter().round_to_pixel(resize_x);
+            let resize_x = side.opposite().side_x(rect);
+            let resize_x = ui.painter().round_to_pixel_center(resize_x);
```

Comment:
It should maybe be noted somewhere that this only makes sense for odd-width strokes (e.g. width=1, width=3 etc)